### PR TITLE
Add:  Node structure

### PR DESCRIPTION
**Pull request by yoLUkAsss**
This should be useful as an example for entities structuring.

### CODE REVIEW

File: jest.config.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  testEnvironment: 'node'
```
Comment (FedeLochbaum) [inline jest.config.js]: ``` export default { testEnvironment: 'node' }  ```

File: src/schema/models.js [hunk 2]
```diff
@@ -1,8 +1,20 @@
 import mongoose from 'mongoose'
-import User from './userSchema'
+import * as Models from './index'
```
Comment (FedeLochbaum) [inline src/schema/models.js]: you can only do ` import Models from './index' `

File: src/schema/node/node.spec.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        Assert.ok(err)
+      })
+    })
+
+  })
+
+
+
+  // it('requesting /api gives a status ok', () => {
+  //   NodeModel.default.create({
```
Comment (FedeLochbaum) [inline src/schema/node/node.spec.js]: please, remove this lines

File: src/server.js [hunk 4]
```diff
@@ -1,12 +1,25 @@
 import express from 'express'
-import api from './routes/api'
-import models, { connectDb } from './schema/models'
+// import models, { connectDb } from './schema/models'
```
Comment (FedeLochbaum) [inline src/server.js]: same message as above

File: src/server.js [hunk 5]
```diff
@@ -1,12 +1,25 @@
...
+// import nodeRoutes from './routes/node'
+import connectionDB from './schema/models'
+
+const DEFAULT_PORT = process.env.PORT || 8080
 
 const app = express()
 
-app.use('/api', api)
+// app.use('/node', nodeRoutes);
```
Comment (FedeLochbaum) [inline src/server.js]: this should be uncommented right ?

File: jest.config.js (hunk 1 above)
Comment (yoLUkAsss) [reply jest.config.js]: I like this better, because is standard among all the *.config.js that exists

File: src/schema/models.js (hunk 2 above)
Comment (yoLUkAsss) [reply src/schema/models.js]: this import should not be used here anymore

File: src/schema/node/node.spec.js (hunk 3 above)
Comment (yoLUkAsss) [reply src/schema/node/node.spec.js]: Ok

File: src/server.js (hunk 4 above)
Comment (yoLUkAsss) [reply src/server.js]: I leave that there to use it as an example on how to use router, but ok

File: src/server.js (hunk 5 above)
Comment (yoLUkAsss) [reply src/server.js]: Correct

File: src/schema/README.md [hunk 6]
```diff
@@ -0,0 +1,9 @@
+
+
+Why we models???
+
+[https://theholmesoffice.com/mongoose-connection-best-practice/](A good practice)
```
Comment (LaimeJesus) [inline src/schema/README.md]: be careful that guide is from 2013, Should we use a updated one?

Comment (yoLUkAsss) [reply src/schema/README.md]: esquiusmi?

Comment (LaimeJesus) [reply src/schema/README.md]: O.O
https://www.youtube.com/watch?v=3D5tkAUNFa4

**Pull request merged**